docker-compose.yml

```diff
@@ -50,6 +51,19 @@ services:
     volumes:
       - ./config/resources:/config
 
+  push-service:
+    build: ../mu-push-service
```

Comment:
Use `image: tenforce/mu-push-service` instead. I know the image is not available yet but it will be at some point.